### ISSUE TITLE
Archiving work and discussion

### ISSUE THREAD

**Issue by npdoty** [w3c/using-github#12]
From the questions thread on the [Guide wiki page](https://www.w3.org/wiki/Guide/GitHub#How_do_we_ensure_archiving_of_work_done_on_GitHub.3F):

> As usual, publication of Working Drafts and Recommendations into w3.org/TR/ will be done by your Team Contact by copying snapshots which satisfy pubrules into the appropriate space, with W3C-guaranteed archiving.
> 
> A common use case would be for editors' drafts to be pointers to the head revision of a git repository's master branch as it appears on GitHub. Because Git itself decentralizes archiving of every change (every user clones all history), backups of version history of Git repositories should be straightforward. W3C systeam is considering automated W3C backups of repositories within the GitHub W3C organization.
> 
> Issues, wikis, pull requests and other GitHub-hosted workflows may not be as straightforward to archive, and your group members may need their own GitHub accounts to make effective use of those features. GitHub repository wikis are themselves Git repositories, so backup should be relatively simple.

We could have separate issues for archiving of each of these (repo, issues, wikis, pull requests), I suppose. My biggest concerns are about backing up the activity that takes place in GitHub issues and pull requests, so that that content doesn't get lost if GitHub goes away and so that we can make commitments to all our work being permanently and publicly archived.

**Comment by darobin** [comment]
That's already covered in http://w3c.github.io/modern-tooling/#de-facto-github.

**Comment by npdoty** [comment]
I was hoping for better documentation now for how this should work beyond "wait for development of a GitHub backup tool", although, if that's the plan, let's write that down too. 

Maybe I'm not clear on whether this repo is for tracking issues/documentation for using GitHub for spec work today, or if this is part of Modern Tooling brainstorming, or if this is detailed issues that tooling work could handle for integration with GitHub.

**Comment by darobin** [comment]
Well, hopefully the development of a GH backup tool is going to get the greenlight to happen this year, so that ought to solve itself shortly — hence the pointer :)

This repo is about eventually documenting everything you need to know to use GitHub at the W3C.

**Comment by tripu** [comment]
@npdoty: note that @darobin wrote [that tool](https://github.com/w3c/gh-backup) already.

AFAIK, it does not deal with issues, wikis, PRs or other project &ldquo;metadata&rdquo;&nbsp;&mdash;&nbsp;it's not a _complete_ back-up. It just `git clone`s and `git fetch`es projects in a smart way. So a more comprehensive solution is being discussed within the Systeam, and the broader team.

In the meantime, the documentation @darobin linked to is current.

**Comment by npdoty** [comment]
Oh man, I'm getting _super_ confused as to the purpose of this project/issues list. Are we keeping issues here for what should be done to improve using Github for W3C spec work? @darobin has a tool to clone and fetch source code, but that doesn't cover the other parts described in the issue, so should we close this issue yet? Or is this an issue list for our documentation on how to use Github? In which case, we should close this issue once we have updated documentation in the correct place (index.html in this repo, or the appropriate wiki, or wherever) -- it looks to me like index.html is currently empty of content.

**Comment by tripu** [comment]
@npdoty: my understanding is that this repo is obsolete; it has not been active lately, and in practice [`github.com/w3c/w3c.github.io`](https://github.com/w3c/w3c.github.io) and [`github.com/w3c/modern-tooling`](https://github.com/w3c/modern-tooling) took over.

I'm trying to clean up pending issues here: either we confirm these proposals have been addressed already, or at the very least that they are mentioned as _modern tooling_. Once that is done, I'm going to suggest we abandon this repo and mark it as deprecated to avoid confusions.

**Comment by npdoty** [comment]
Thanks.
